src/lib/core/say.c

```diff
@@ -267,7 +267,7 @@ log_rotate(struct log *log)
 	if (pm_atomic_load(&log->type) != SAY_LOGGER_FILE)
 		return 0;
 
-	ERROR_INJECT(ERRINJ_LOG_ROTATE, { usleep(10); });
+	ERROR_INJECT(ERRINJ_LOG_ROTATE, { usleep(1000); });
```

Comment:
The best solution here will be an .iparam for the injection, with a rename to ERRINJ_LOG_ROTATE_DELAY. Can be a follow-up

Reply:
Should be done separately